app/Http/Controllers/GenreController.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace App\Http\Controllers;
+use App\Models\Genre;
+use Illuminate\Http\Request;
+
+class GenreController extends Controller
+{
+    private Genre $genre;
+
+    public function __construct(Genre $genre) 
+    {
+        $this->genre=$genre;
+    }
+
+    public function getAllGenres(Genre $genre) 
+    {
+        $genres = Genre::all(['id', 'name']);
```

Comment:
Because you have $genre in the constructor, you should be using the $genre property instead of directly acessing Genre here 

```
$this->genre->all()
```

instead of 

```
Genre::all()
```